### ISSUE TITLE
Background spotlight

### ISSUE THREAD

**Issue by yrtnsky** [ORelio/Spotlight-Downloader#29]
In Windows 11 was added a feature to use Spotlight images for wallpapers.
There appear images 3840x2160 size.
Does anyone know what request can be used to get a proper reply?
I guess ID is 88000326, json file looks different from files for lockscreen.

**Comment by ORelio** [comment]
Hi,
For now, I'm not aware of how Windows 11 downloads its wallpaper images.
I tried to replace the ID for spotlight lockscreen image API request with the one you provided but that was not enough.

**Comment by MNLierman** [comment]
I'm going to open another issue, but I wanted to comment on this one. I'm aware that you made this in 2021, so this is a little aged, but still it remains that Spotlight API uses 4k, and this downloads only 1080. On my 4K computers this looks like trash. Naturally, as time has gone on, I have replaced most of my monitors to 4K. My new laptop is 4k, so this script, while cool, is unusable because of how blurry and awful the images look. Perhaps the script is using an old API version, and Microsoft updated it? I'll link some other resources I found that have 4K. This tool is still cool because it randomly sets one of the images to the lockscreen, but I'm sure I could duplicate that. Just wondering if you use this yourself, because it's glaringly obvious how unusable this is on 4K. Maybe I can improve it, but I don't know anything about Spotlight and I'm not well versed with JSON and APIs in general, assuming JSON is involved in the API request. I do know C# really well.
